std_msgs/String name
std_msgs/Int32 object_id
geometry_msgs/Point32[] bounding_box
